community/keyutils/patches/request-key.conf.patch

```diff
@@ -0,0 +1,21 @@
+--- request-key.conf.old	2013-05-28 15:34:10.966001242 +0200
++++ request-key.conf	2013-05-28 15:35:34.857969598 +0200
+@@ -31,11 +31,11 @@
+ 
+ #OP	TYPE	DESCRIPTION	CALLOUT INFO	PROGRAM ARG1 ARG2 ARG3 ...
+ #======	=======	===============	===============	===============================
+-create  dns_resolver *		*               /sbin/key.dns_resolver %k
+-create	user	debug:*		negate		/bin/keyctl negate %k 30 %S
+-create  user    debug:*         rejected        /bin/keyctl reject %k 30 %c %S
+-create  user    debug:*         expired         /bin/keyctl reject %k 30 %c %S
+-create  user    debug:*         revoked         /bin/keyctl reject %k 30 %c %S
+-create	user	debug:loop:*	*		|/bin/cat
++create  dns_resolver *		*               /usr/bin/key.dns_resolver %k
++create	user	debug:*		negate		/usr/bin/keyctl negate %k 30 %S
++create  user    debug:*         rejected        /usr/bin/keyctl reject %k 30 %c %S
++create  user    debug:*         expired         /usr/bin/keyctl reject %k 30 %c %S
++create  user    debug:*         revoked         /usr/bin/keyctl reject %k 30 %c %S
++create	user	debug:loop:*	*		|/usr/bin/cat
+ create	user	debug:*		*		/usr/share/keyutils/request-key-debug.sh %k %d %c %S
```

Comment:
What does this do?

Reply:
If it only changes `/bin` to /usr/bin` it can be removed. They're symlinked.

Reply:
fixes paths to /usr

Reply:
Can be removed then.